frame/vamm/diagrams/function_create.plantuml

```diff
@@ -0,0 +1,16 @@
+@startuml
```

Comment:
I think I'm seeing changes from a previous PR. Can you try merging `escher` into this?

Reply:
Hm, I don't know why this happened...
I had already merged `escher` when I first submitted the PR :thinking: 

Anyways, I merged it again and now I think you won't have this problem anymore. Thanks for the input :+1: